src/Symfony/Component/Security/Http/FirewallMapInterface.php

```diff
@@ -21,16 +21,19 @@
 interface FirewallMapInterface
 {
     /**
-     * Returns the authentication listeners, and the exception listener to use
-     * for the given request.
+     * Returns the authentication listeners, the exception listener and the logout
+     * listener to use for the given request.
      *
      * If there are no authentication listeners, the first inner array must be
      * empty.
      *
      * If there is no exception listener, the second element of the outer array
      * must be null.
      *
-     * @return array of the format array(array(AuthenticationListener), ExceptionListener)
+     * If there is no logout listener, the third element of the outer array must
+     * be null.
+     *
+     * @return array of the format array(array(AuthenticationListener), ExceptionListener, LogoutListener)
```

Comment:
@nicolas-grekas are we fine with such change on 2.7?

Reply:
existing code should be just fine ignoring the 3rd value, so looks ok to me